### PR TITLE
data/kavyamadasu.json

### PR DESCRIPTION
**Pull request by kavyamadasu**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4212"></a>

### CODE REVIEW

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please keep your json file inside the data folder

File: kavyamadasu [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "url": "https://twitter.com/kavya_madasu",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/kavyamadasu/",
+      "icon": "FaGithub"
+    },{
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/kavyamadasu-782337171",
```
Comment (Aadarsh805) [inline kavyamadasu:18]: fix this link to `https://www.linkedin.com/in/kavyamadasu-782337171`

File: kavyamadasu [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Kavya Madasu",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Iam a full stack Java Developer"
```
Comment (Aadarsh805) [inline kavyamadasu:5]: missing comma at the end of line 5

File: kavyamadasu [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/kavyamadasu/",
+      "icon": "FaGithub"
+    },{
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/kavyamadasu-782337171",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline kavyamadasu:19]: change this LinkedIn icon to `FaLinkedin`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
change your file name to `data/kavyamadasu.json`

**Review by kumarsonsoff3 — DISMISSED** [review]
Please add your file to data directory & add `.json` as file extension. 🛠

To move your file to data directory, you need to rename your [file here ](https://github.com/kavyamadasu/LinkFree/edit/kavyamadasu-patch-1/kavyamadasu)to `data/kavyamadasu.json`. 

Change `kavyamadasu`  - > `data/kavyamadasu.json` Let us know if you need any help with this! 

image

**Pull request merged**